src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -71,6 +75,11 @@ public void testInitialize() {}
 	@Override
 	public void testExecute() {}
 
+	byte[] matchConfigurationCANData() {
+		return new byte[] {(byte) DriverStation.getInstance().getAlliance().ordinal(),
```

Comment:
Considering that three years ago was when they rewrote the entire protocol and changed to the RoboRIO, this is well within the lifetime design of our code.